pipeline/lovd/combineEquivalentVariantSubmissions.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+
+"""
+Zack Fischmann
+8/14/2018
+
+Description:
+    Combine LOVD variants considered to be equivalent submissions.
+"""
+
+import argparse
+import csv
+
+def mergeRows(oldRow, newRow):
```

Comment:
Some small unit test would have been nice here ;)

Also pandas might have helped here (although it might have been a tad less straightforward)

Reply:
added tests in https://github.com/BRCAChallenge/brca-exchange/pull/904/files#diff-e20b04b4d54db2ca8f08b7671d771df5

Reply:
I opted to leave this as is instead of convert to pandas